execution_engine2.spec

```diff
@@ -122,16 +160,24 @@
             Meta meta;
             int wsid;
             string parent_job_id;
+            JobRequirements job_requirements;
+            boolean as_admin;
         } RunJobParams;
```

Comment:
I considered making a new endpoint so that the job requirements weren't embedded in `RunJobParams` but this way seems less nasty.